src/tests/ut/reserve_queue/reserve_queue_test.cpp

```diff
@@ -0,0 +1,412 @@
+#include "reserve_queue_test.hpp"
+
+#include "gmock/gmock.h"
+
+namespace
+{
+using ::testing::ElementsAre;
+using ::testing::UnorderedElementsAre;
+
+class AddAwaitingPatientShould : public ReserveQueueFixture
```

Comment:
What was the reasoning to introduce another layer of fixture?

Reply:
Adding another layer is my trick to provide better explanatory names for tests.
Inheriting from base fixture and providing new name allows me to use all fixture functionalities and have test print like this:
![image](https://github.com/coders-school/desktop-business-app/assets/82645568/fd61aee0-f334-478f-a13c-8fd354dbd2b4)
or
![image](https://github.com/coders-school/desktop-business-app/assets/82645568/0d20d9f4-b181-4227-8fe8-e11a566f1269)

In general it allows me to have name of the function printed following with expectation after the dot.